Doc/library/os.path.rst

```diff
@@ -344,22 +344,29 @@ the :mod:`glob` module.)
       Accepts a :term:`path-like object`.
 
 
-.. function:: realpath(path)
+.. function:: realpath(path, *, strict=False)
 
    Return the canonical path of the specified filename, eliminating any symbolic
    links encountered in the path (if they are supported by the operating
    system).
 
-   .. note::
-      When symbolic link cycles occur, the returned path will be one member of
-      the cycle, but no guarantee is made about which member that will be.
+   If the path doesn't exist and *strict* is ``True``, :exc:`FileNotFoundError`
+   is raised. If *strict* is ``False``, the path is resolved as far as possible
```

Comment:
This means the target path, right? If `path` doesn't exist there'll always be an exception regardless of `strict`.

Reply:
In non-strict mode, `os.path.realpath()` doesn't need _any_ part of the path to exist at _any_ stage. This doesn't just apply to broken symlink targets.

```python
>>> os.path.realpath('/foo/bar/baz')
'/foo/bar/baz'
```

Reply:
FYI this is a straight copy-paste of the description of `strict` for `pathlib.Path.resolve()`